ArduCopter/mode_auto.cpp

```diff
@@ -1551,7 +1586,35 @@ bool Copter::ModeAuto::verify_land()
     switch (land_state) {
         case LandStateType_FlyToLocation:
             // check if we've reached the location
+#if FRAME_CONFIG == HELI_FRAME
+            if (wp_nav->use_l1_navigation()) {
+            if (copter.wp_nav->reached_l1_destination()) {
```

Comment:
indent?

Reply:
Tom, we need L1 Nav for heli. Helicopters are considerably more high-performance machines than multicopters and they do not perform well on Copter Nav. As it is, the attitude controller needs some work for heli because it does not understand that nose up means climb, and nose down means dive for a helicopter. Helicopters only share common traits with multicopters in hover. Once they enter translational lift speed they become like an airplane with the only difference being the airplane has a fixed wing, the helicopter has a rotary wing. This is why L1 Nav fits so well for helicopters.